filebeat/docs/filebeat-general-options.asciidoc

```diff
@@ -73,6 +73,26 @@ NOTE: Filtering out a huge number of logs can cause many registry updates, slowi
 down processing. Setting `registry.flush` to a value >0s reduces write operations,
 helping Filebeat process more events.
 
+[float]
+==== `registry.migrate_file`
+
+Prior to Filebeat 7.0 the registry is stored in one file only. On startup
+Filebeat will automatically migrate the old Filebeat 6.x registry file to the 
+new directory based registry.
+
+By default the file location to migrate from is
+assumed to be given by `filebeat.registry.path`, but in case the path has been
+changed when upgrading one can use `filebeat.registry.migrate_file`.
+
+[source,yaml]
+-------------------------------------------------------------------------------------
+filebeat.registry.path: ${path.data}/registry
+filebeat.registry.migrate_file: /path/to/old/registry_file
+-------------------------------------------------------------------------------------
+
+Filebeat will use `filebeat.registry.migrate_file` only if the regisry does not
```

Comment:
This is a little ambiguous. Are you saying that the file will only be migrated if a registry that uses the directory format does not yet exist?

Reply:
Yes. When migrating, the old registry file will be moved into it's new location.